readme.md

```diff
@@ -33,6 +36,14 @@ The destination folder is optional, and defaults to the current folder name. Wat
     apis      programs
     github readme.md
 
+#### Adding Authentication
+To use authenticated requests you must first [create a github](https://help.github.com/articles/creating-an-access-token-for-command-line-use/) api token on your github account.
+
+    github auth <user> <token> [-D <username>]
```

Comment:
Should be `github auth <user> [<token>| -d]`  in my opinion. The implementation too, obviously. Printing out the token with just the user argument would be handy too